server/java/src/main/java/com/godaddy/asherah/grpc/ServerApp.java

```diff
@@ -34,6 +62,12 @@
   @CommandLine.Option(names = "--jdbc-url", defaultValue = "${env:ASHERAH_CONNECTION_STRING}",
       description = "JDBC URL to use for JDBC metastore. Required for JDBC metastore.")
   private static String jdbcUrl;
+  @CommandLine.Option(names = "--enable-key-suffix", defaultValue = "${env:ASHERAH_ENABLE_KEY_SUFFIX}",
+      description = "Configure the metastore to use key suffixes (only supported by DYNAMODB)")
+  private String keySuffix;
```

Comment:
should be static

Reply:
I think none of the options need to be `static` except the ones in the inner classes. Would require a change either way. Can discuss offline